change-validation/src/main/scala/com/normation/plugins/changevalidation/snippet/ChangeRequestDetails.scala

```diff
@@ -167,12 +170,20 @@ class ChangeRequestDetails extends DispatchSnippet with Loggable {
   }
 
   def displayActionButton(cr: ChangeRequest, step: WorkflowNodeId): NodeSeq = {
-    val authz   = CurrentUser.getRights.authorizationTypes.toSeq.collect { case right: ActionType.Edit => right.authzKind }
+    val authz   = Nil // we are sideStepping it, see: https://issues.rudder.io/issues/22595
```

Comment:
this is the problem: we don't look for `any` because we are comaring on strings, not with the correct `Authorization` method. 
We can change that now, see: https://issues.rudder.io/issues/22595